src/model/queries/addUser.js

```diff
@@ -0,0 +1,14 @@
+const db_connection = require('./../database/db_connection');
+
+const addUser = (userObject, callback) => {
+  const sql = {
+    text: 'insert into users (user_name,user_email,user_password,permission)values($1,$2,$3,$4)',
+    values: [userObject.username, userObject.email, userObject.password, 'patient'],
+  };
+  db_connection.query(sql, (err, result) => {
```

Comment:
Kannan Please change the callback to promises like the rest of us :smile_cat:

Reply:
i will try .. :+1: